content/en/examples/application/deployment-sidecar.yaml

```diff
@@ -21,10 +21,8 @@ spec:
           volumeMounts:
             - name: data
               mountPath: /opt
-      initContainers:
-        - name: logshipper
+        - name: sidecar-container
```

Comment:
This change is not correct. 

The current examples on the page depict the use of InitContainers acting as sidecars using the feature gate named SidecarContainers.

The suggestion [here](https://github.com/kubernetes/website/issues/45406#issuecomment-1973466477) is to add a new section to the page depicting the legacy way of defining sidecars. 

Do not change the existing examples.

Reply:
Sure. It seems I missed this.
I've add two new yaml example within folder application/